cdl_examples/tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803.cdl

```diff
@@ -0,0 +1,90 @@
+netcdf tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803 {
+dimensions:
+	axis_nbounds = 2 ;
+	x = 567 ;
+	y = 264 ;
+	deptht = 75 ;
+	time = UNLIMITED ; // (1 currently)
+variables:
+	float nav_lat(y, x) ;
+		nav_lat:standard_name = "latitude" ;
+		nav_lat:long_name = "Latitude" ;
+		nav_lat:units = "degrees_north" ;
+	float nav_lon(y, x) ;
+		nav_lon:standard_name = "longitude" ;
+		nav_lon:long_name = "Longitude" ;
+		nav_lon:units = "degrees_east" ;
+	float deptht(deptht) ;
```

Comment:
Should be `depth(depth)` (no t at the end) https://github.com/WCRP-CORDEX/cordex-cmip6-cmor-tables/blob/508ca0ee5df5c27663622371bbf34f0ba916970b/Tables/CORDEX-CMIP6_coordinate.json#L233-L254